tck/rest/how_to_run_tests.adoc

```diff
@@ -1,23 +1,5 @@
 = How to run the tests
 
-== Server under test
-Start your Eclipse Microprofile Server with Metrics enabled
+== Tests 
+TCK is currently under development
 
```

Comment:
I wouldn't remove all of the text indicating how to run the tests.  Just include the "note" that indicates that the TCK execution is still under development.